domains/sam.json

```
@@ -0,0 +1,11 @@
+{
+  "description": "Sam Poder's personal website.",
+  "repo": "https://github.com/sampoder/w2",
+  "owner": {
+    "username": "sampoder",
+    "email": "sam@sampoder.com"
+  },
+  "record": {
+    "CNAME": "sampoder.com"
```

Unfortunately, vercel doesn't support our subdomain service. Since, you already have a dot com, a URL redirection to your website would be a better choice anyway.

```suggestion
    "URL": "https://sampoder.com"
```

---

great idea, thank you @phenax!